src/index.ts

```diff
@@ -1,11 +1,12 @@
 import { Observable } from "rxjs/Observable";
 import { IpcMark } from "./common/types";
 
-import mainProcessMonitor from "./main";
-import rendererProcessMonitor from "./renderer";
-
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
How do you feel about using dynamic imports here instead? It might make it a little funkier, but it would eliminate the need to disable an eslint rule. 

the funky part: doing so would give us a `Promise<Observable<IpcMark>>`. So we'd need to convert it to `Observable<IpcMark>` via:
```
import { from } from "rxjs/observable/from";
import "rxjs/add/operator/mergeAll";

const flattenFromPromise = <T>(p: Promise<Observable<T>>): Observable<T> => from(p).mergeAll()
```

Reply:
@KeijiBranshi , using `await import(...)` would make `ipcMonitor` available only asynchronously. Of course maybe I don't fully understand your suggestion, can you elaborate?

Reply:
ipcMonitor is already of type `Observable` so everything it emits is already asynchronous. Using `await import(...)` would just wrap it as a `Promise<Observable>` which can be _synchronously_ flattened back down to `Observable` using the above code sample: `flattenFromPromise(ipcMonitor)`

Reply:
I think of it like the equivalent of using `.then()` to flatten nested promises. But in this case we're flattening nested `Observable`s (in rx, `Promise` is sort of a special case of `Observable`)

<img width="777" alt="image" src="https://user-images.githubusercontent.com/10525281/96914514-f6c9d780-1459-11eb-9e95-abc33e1514a4.png">

Reply:
Awesome. Updated.